### PR TITLE
Added data for professional experience

### PR DESCRIPTION
**Pull request by ayu-ayushisood**
• Added professional experience data in json format.
• Retrieved json data and integrated in Experiences screen.
• Made only right section scrollable, so that left sidebar remains

### CODE REVIEW

File: src/components/Company/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <div className="company-info">
+            <div className="company-logo">
+                <img src={props.logo} />
+            </div>
+            <div className="details-section">
+                <div className="title">{props.title}</div>
+                <div>{props.subtitle}</div>
+                <div>{props.companyName}</div>
+                <div className="roles">Roles and Responsibilities</div>
+                {props.roles}
```
Comment (karanasthana) [inline src/components/Company/index.js:28]: why roles here as well as on line 34 ?

File: src/components/Company/index.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                {isMilestone &&
+                    <>
+                        <div className="title">{props.milestone.title}</div>
+                        <div>{props.milestone.heading}</div>
+                        <div className="roles">Roles and Responsibilities</div>
+                        {getRoles(props.milestone.roles)}
```
Comment (karanasthana) [inline src/components/Company/index.js]: maybe assign different names (both getRoles are a bit confusing)

File: src/components/Company/index.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './style.css';
+
+const Company = (props) => {
+
+    const isMilestone = Object.entries(props.milestone).length > 0;
```
Comment (karanasthana) [inline src/components/Company/index.js:6]: Nice 👍🏻

File: src/common/experienceData.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "data": [
+        {
+            "id": 1,
+            "name": "Magic EdTech",
+            "place": "Noida, India",
+            "logo": "https://ik.imagekit.io/zw5zu6q4kxa/magic_4d1UjcP2s.png",
+            "title": "Software Engineer",
+            "heading": "Sr. Associate (September 2020 - Present)",
+            "roles": [
+                "Working extensively in creating web based ed-gaming platforms in Reactjs.", "Leading the tech team", "Responsible for the agile development", "Working on building future proof e-learning games from kindergaten to 8th grade students.", "Extensively working with animations and overall state management."
```
Comment (karanasthana) [inline src/common/experienceData.json]: Can these different roles be in different lines?

That will increase coder readability

File: src/components/Company/index.js (hunk 1 above)
Comment (ayu-ayushisood) [reply src/components/Company/index.js:28]: 34 is the milestone's role. means the milestone in KC (internship, then FTE - FE dev). So, the roles are different for both

File: src/common/experienceData.json (hunk 4 above)
Comment (ayu-ayushisood) [reply src/common/experienceData.json]: done

File: src/components/Company/index.js (hunk 1 above)
Comment (karanasthana) [reply src/components/Company/index.js:28]: han toh specify that in the variable name na ?

**Pull request merged**